ga4gh/frontend.py

```diff
@@ -195,13 +206,33 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     theBackend.setDefaultPageSize(app.config["DEFAULT_PAGE_SIZE"])
     theBackend.setMaxResponseLength(app.config["MAX_RESPONSE_LENGTH"])
     app.backend = theBackend
+    app.secret_key = os.urandom(24)
+    app.oidcClient = oic.oic.Client()
+    if oidcConfig:
+        with open(oidcConfig) as oidcConfigFile:
+            configDict = yaml.load(oidcConfigFile)
+        app.oidcClient.redirect_uris = [configDict['redirecturi']]
+        app.oidcClient.provider_config(configDict['provider'])
+    if 'registration_endpoint' in app.oidcClient.provider_info:
```

Comment:
This is brittle here --- if oidcConfig isn't provided,  `app.oidcClient.provider_info` is None and we get an exception.